hack/kubelet-plugin-prestart.sh

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env bash
+
```

Comment:
> `hack/kubelet-plugin-prestart.sh`

Do we know of a better place to put this in the repo?

Reply:
Not as of now. Where does the operator put such scripts?

Reply:
I will leave it here for now

Reply:
> Where does the operator put such scripts?

There is a `scripts` top-level dir: https://github.com/NVIDIA/gpu-operator/tree/main/scripts

Maybe we should create that. I like that better.

Reply:
can do another time, leaving this as-is for now